client/me/help/gm-closure-notice/index.jsx

```diff
@@ -0,0 +1,176 @@
+
+/**
+ * Internal dependencies
+ */
+import { isEcommercePlan, isBusinessPlan, isPremiumPlan, isPersonalPlan } from 'lib/plans';
+import FoldableCard from 'components/foldable-card';
+import FormSectionHeading from 'components/forms/form-section-HEADING';
+import { useLocalizedMoment } from 'components/localized-moment';
+import { getSitePlanSlug } from 'state/sites/selectors';
+import { getHelpSelectedSiteId } from 'state/help/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const DATE_FORMAT_SHORT = 'MMMM D';
+const DATE_FORMAT_LONG = 'dddd, MMMM Do';
+
+const GMClosureNotice = ( {
+	businessAndEcommerceClosesAt,
+	businessAndEcommerceReopensAt,
+	compact,
+	defaultClosesAt,
+	defaultReopensAt,
+	displayAt,
+	selectedSitePlanSlug,
+} ) => {
+	const translate = useTranslate();
+	const moment = useLocalizedMoment();
+
+	const isBusinessOrEcommercePlan =
+		isBusinessPlan( selectedSitePlanSlug ) || isEcommercePlan( selectedSitePlanSlug );
+	const isPersonalOrPremiumPlan =
+		isPersonalPlan( selectedSitePlanSlug ) || isPremiumPlan( selectedSitePlanSlug );
+	const isNonPlan = ! isBusinessOrEcommercePlan && ! isPersonalOrPremiumPlan;
+
+	const currentDate = moment();
+	const guessedTimezone = moment.tz.guess();
+
+	const [ closesAt, closedUntil, reopensAt ] = [
+		moment.tz(
+			isBusinessOrEcommercePlan ? businessAndEcommerceClosesAt : defaultClosesAt,
+			guessedTimezone
+		),
+		moment
+			.tz(
+				isBusinessOrEcommercePlan ? businessAndEcommerceReopensAt : defaultReopensAt,
+				guessedTimezone
+			)
+			.subtract( 1, 'days' ),
+		moment.tz(
+			isBusinessOrEcommercePlan ? businessAndEcommerceReopensAt : defaultReopensAt,
+			guessedTimezone
+		),
+	];
+
+	if ( ! currentDate.isBetween( displayAt, reopensAt ) ) {
+		return null;
+	}
+
+	const HEADING = translate( 'Limited Support %(closesAt)s – %(closedUntil)s', {
+		args: {
+			closesAt: closesAt.format( DATE_FORMAT_SHORT ),
+			closedUntil: closedUntil.format(
+				closedUntil.isSame( closesAt, 'month' ) ? 'D' : DATE_FORMAT_SHORT
+			),
+		},
+	} );
+
+	const mainMessageArgs = {
+		closesAt: closesAt.format( DATE_FORMAT_LONG ),
+		closedUntil: closedUntil.format( DATE_FORMAT_LONG ),
+		reopensAt: reopensAt.format( DATE_FORMAT_LONG ),
+	};
+
+	const MAIN_MESSAGES = {
+		before: {
+			hasPlan: translate(
+				'Live chat support will be closed %(closesAt)s through %(closedUntil)s, included. Email support will be open during that time, and we will reopen live chat on %(reopensAt)s.',
+				{ args: mainMessageArgs }
+			),
+			nonPlan: translate(
+				'Private support will be closed %(closesAt)s through %(closedUntil)s, included. We will reopen private support on %(reopensAt)s.',
+				{ args: mainMessageArgs }
+			),
+		},
+		during: {
+			hasPlan: translate(
+				'Live chat support will be closed through %(closedUntil)s, included. Email support will be open during that time, and we will reopen live chat on %(reopensAt)s.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Live chat support will be closed for the Thanksgiving holiday on %(closed_start_date)s. Email support will be open during that time, and we will reopen live chat on %(closed_end_date)s.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).